src/main/kotlin/mate/academy/LetterRemover.kt

```diff
@@ -1,5 +1,10 @@
 package mate.academy
 
-class LetterRemover {
-    // implement ...
+object LetterRemover {
+        @JvmStatic
+        fun removeLetter(input: String): CharArray {
+            return input.lowercase().map {
```

Comment:
This is fine, however, please check this function
https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/replace.html

Reply:
I don't know why, but if I make a class, the checkstyle does not pass and says: The class LetterRemover only contains utility functions. Consider defining it as an object. [UtilityClassWithPublicConstructor]
That's why I make an object!

Reply:
It's checkstyle plugin behaviour applied for this particular case. Nevermind